### ISSUE TITLE
Edgium support

### ISSUE THREAD

**Issue by RussKie** [berzniz/github_pr_tree#94]
Thank you for the extension, it made my life heaps easier!

However I am now encouraged to migrate to Edgium from Chrome, and the extension doesn't appear to be working.
![image](https://user-images.githubusercontent.com/4403806/75410983-c89b0300-5971-11ea-919c-e9cb97547874.png)

**Comment by RussKie** [comment]
I just noticed it stopped working in Chrome today, and suspect it may be due to some markup tweaks on the GitHub side.

I've noticed, for instance, `id`s for files have changed. E.g. it used to be (I had an old tab opened):
```html
<div id="diff-6" class="file ...." data-codeowners="...." data-file-type=".cs" data-file-deleted="false">
   ....
</div>
```
and now it is
```html
<div id="diff-56f8482aeb786dd105fbf93dedf2615f" class="file ...." data-codeowners="..." data-file-type=".targets" data-file-deleted="false">
   ....
</div>
```

May be something else has changed which broken the extension 🤔

**Comment by berzniz** [comment]
Thanks for reporting. I will issue an update today or tomorrow

**Comment by berzniz** [comment]
A fix will be released in version 1.0.126 (later today).

The issue was fixed in #96 - you can pull the changes and continue working on Edge. It should work for Edge as well I believe

**Comment by RussKie** [comment]
Awesome! Thank you
